src/docs/welcome/quickStart/DOC.md

```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
 1. Run the container with default options
 
     ```sh
-    bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh
     ```
 
-#### More options for running
+##### More options for running
 
 * Run with the local Legacy Insighs API
 
     ```sh
-    LOCAL_API=true bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh LOCAL_API=true
```

Comment:
Running with chrome is buried in the docs right?
One thing I see tons of teams doing is running chrome locally when they dont need to.
The Chrome docs are really for us, perhaps we should just put them in a separate area/file.

Reply:
I'll just remove them for now.